### ISSUE TITLE
Change file name of lcovonly report

### ISSUE THREAD

**Issue by Ritzlgrmft** [monounity/karma-typescript#61]
I generate the code coverage file using 

```
reports:
{
  "lcovonly": "coverage"
}
```

This generates the file `coverage/&lt;browser>/lcovonly`. Unfortunately, `codecov` does not recognize this file. But when I rename it to `lcov.info`, `codecov` is fine.

For me, it would be great, if I could generate the `lcov.info` file directly.

**Comment by erikbarke** [comment]
Hey @Ritzlgrmft, this was fixed in #56, you should now be able to configure reports like this:

```javascript
karmaTypescriptConfig: {
    reports:
    {
        "lcovonly": {
            "directory": "coverage",
            "filename": "lcovonly/lcov.info"
        }
    }
}
```

You can try it out if you install from the latest dev branch, ie `npm install https://github.com/monounity/karma-typescript.git#2.1.6`.
If you’re behind a firewall and can’t install from GitHub, you can download the branch as a zip and install manually.

The latest documentation: https://github.com/monounity/karma-typescript/tree/2.1.6

**Comment by Ritzlgrmft** [comment]
It works.

Thanks for the quick response!
